spec/controllers/news_controller_spec.rb

```diff
@@ -67,4 +67,82 @@
       end
     end
   end
+
+
+  describe 'GET #index with filters' do
+    let!(:department_a) { Fabricate(:department) }
+    let!(:department_b) { Fabricate(:department) }
+    let!(:minister_c) { Fabricate(:minister) }
+    let!(:minister_d) { Fabricate(:minister) }
+
+    let!(:a_home_node) { Fabricate(:section_home, section: department_a) }
+    let!(:b_home_node) { Fabricate(:section_home, section: department_b) }
+    let!(:c_home_node) { Fabricate(:section_home, section: minister_c) }
+    let!(:d_home_node) { Fabricate(:section_home, section: minister_d) }
+
+    let!(:article_a) { Fabricate(:news_article, state: 'published', section: department_a, sections: [department_a]) }
```

Comment:
Did you want to update the fabricator to automatically add the section to the sections array?